modules/standard/Time.chpl

```diff
@@ -72,39 +72,13 @@ module Time {
                                            out yday:    int(32),
                                            out isdst:   int(32));
 
-  /* Begin day-of-week enums */
-
-  /* Controls whether :type:`dayOfWeek` starts with `Monday = 0` or
-  `Monday = 1`.
-
-    - If true, :type:`dayOfWeek` represents Monday as day 1.
-    - If false, :type:`dayOfWeek` represents Monday as day 0. This behavior is
-      deprecated and will be removed in an upcoming release.
-
-    The deprecated behavior is on by default. To opt-in to the new behavior,
-    recompile your program with ``-scIsoDayOfWeek=true``.
-
-  */
-  config param cIsoDayOfWeek = false;
```

Comment:
I think we've been keeping these deprecation-related config params around for one extra release and deprecating them after they stop being used. This way if a user has `-scIsoDayOfWeek=true` in their makefile etc, they get a deprecation warning instead of a "no such param" warning.

It's possible that this flag is obscure enough that we don't need to do that tho

Reply:
Oh, yeah I should go ahead and do that